src/index.ts

```diff
@@ -482,6 +547,7 @@ const transform = async ({
     platform: "neutral",
     define: {
       __CONFIG__: JSON.stringify(config),
+      __BASE_PATH__: JSON.stringify(basePath),
```

Comment:
is the `JSON.stringify` needed?

Reply:
I'm not a 100% certain. Happy to change it if ESBuild won't inline the string literal as a literal

Reply:
I must say that I don't really get it... 😕 

but it looks like that in other places we also run `JSON.stringify` on a string:
![Screenshot 2023-03-10 at 21 42 44](https://user-images.githubusercontent.com/61631103/224434003-3c1a0dc7-50b2-4e74-83c3-db5b0711f852.png)

So I think that this is fine (and consistent) and we can revisit this later 🙂👍

Reply:
> but it looks like that in other places we also run `JSON.stringify` on a string

Just to add, the reason for the JSON.stringify on chunkFilePath is so that the raw value is `"'...'"` instead of `"..."` for the AST. You can see this demonstrated if you put in some code that uses require on astexplorer.net.

Reply:
ah yeah I see, sorry yeah it slipped my mind that JSON.stringify adds quotes! thanks! 🙂